### DISCUSSION TITLE
Mini bug pas méchant sur les sondages

### DISCUSSION THREAD

**Discussion by Seikame** [fibreville/minotaure_php#87] [Parlons peu, parlons dev]
J'ai répété le cas 2 fois pour vérifier parce que ça me semblait bizarre : **Lorsqu'on vote pour un sondage déjà clôturé** (me jugez pas D: ) **ça supprime la possibilité de voter pour le sondage suivant**, ça se résout sur les sondages ultérieurs. 

Rien de bien méchant, ça n'arrivera qu'aux gens qui sont pas au taquet et qui cliquent machinalement sur le vote au lieu de recharger la page ou attendre qu'elle s'actualise. Et ça ne les réduira au silence que pour le sondage d'après. Mais au cas où ça puisse être résolu en 30 secondes, je me permets de le mentionner. :p

**Comment by uurshin** [comment]
Intéressant ! Ca s'explique assez facilement dans le code, je pense savoir comme résoudre ça facilement.
J'avais effectivement eu des remontées dans ce sens mais aucune vraie confirmation donc merci !
